INCHI-1-TEST/install.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-apt update && apt install -y python3-pip
+apt update && apt install -y python3-pip cmake
```

Comment:
Instead of adding another Dockerfile, I suggest to install `cmake` into the existing devcontainer.

Reply:
Yeah, I added a minimalistic one because the one below gained test specific stuff that I thought you don't need to keep while working on the code; if that's not true feel free skip the addition

https://github.com/IUPAC-InChI/InChI/blob/34c4d0fd4510031ea5c5a23f1f98271365959719/INCHI-1-TEST/Dockerfile#L1-L46